model/user.go

```diff
@@ -191,13 +191,29 @@ func (u *User) SetDefaultNotifications() {
 	u.NotifyProps["desktop"] = USER_NOTIFY_ALL
 	u.NotifyProps["desktop_sound"] = "true"
 	u.NotifyProps["mention_keys"] = u.Username + ",@" + u.Username
-	u.NotifyProps["first_name"] = "false"
 	u.NotifyProps["all"] = "true"
 	u.NotifyProps["channel"] = "true"
-	splitName := strings.Split(u.Nickname, " ")
-	if len(splitName) > 0 && splitName[0] != "" {
+
+	if u.FirstName == "" {
+		u.NotifyProps["first_name"] = "false"
+	} else {
 		u.NotifyProps["first_name"] = "true"
-		u.NotifyProps["mention_keys"] += "," + splitName[0]
+		u.NotifyProps["mention_keys"] += "," + u.FirstName
```

Comment:
Yeah, I can remove that line. It's actually unnecessary since it only adds the first name to that list as part of setting the default notifications for a user. It's not currently even possible to have a first name when those are set.